Makefile

```diff
@@ -1,16 +1,18 @@
-REGISTRY ?= quay.io
-ORG ?= rh-nfv-int
+VERSION         := 0.2.5
+TAG             := v$(VERSION)
+REGISTRY        ?= quay.io
+ORG             ?= rh-nfv-int
 DEFAULT_CHANNEL ?= alpha
-
-CONTAINER_CLI ?= podman
-CLUSTER_CLI ?= oc
+CONTAINER_CLI   ?= podman
+CLUSTER_CLI     ?= oc
+OPERATOR_NAME   := cnf-app-mac-operator
 
 # VERSION defines the project version for the bundle.
 # Update this value when you upgrade the version of your project.
 # To re-generate a bundle for another specific version without changing the standard setup, you can:
 # - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
-VERSION ?= 0.2.5
+
```

Comment:
Let's remove all these comments, I think `VERSION` is clear on what the purpose of it is.

Reply:
Fixed, thanks!